pyvista/_vtk.py

```diff
@@ -545,5 +547,5 @@ def lazy_vtkCameraOrientationWidget():
     try:
         from vtkmodules.vtkInteractionWidgets import vtkCameraOrientationWidget
     except ImportError:  # pragma: no cover
-        raise ImportError('vtkCameraOrientationWidget requires vtk>=9.1.0')
+        raise VTKVersionError('vtkCameraOrientationWidget requires vtk>=9.1.0')
     return vtkCameraOrientationWidget()
```

Comment:
Thanks for the great PR! Very good first contribution :)

There seems to be a circular import somewhere here. If you just start up python and try to `import pyvista` you should see an error. I think you can fix it if you move the `from pyvista.core.errors` import into the function, perhaps inside the `except` even. You'll be able to check: if `import pyvista` starts working then this was it :)

Do let us know if you need any help.

Reply:
Resolved!